client/luigi-client.js

```diff
@@ -431,6 +431,15 @@
          */
         removeBackdrop: function removeBackdrop() {
           window.parent.postMessage({ msg: 'luigi.remove-backdrop' }, '*');
+        },
+        /**
+         * Reloads the environment list in the Context Switcher
```

Comment:
```suggestion
         * Reloads the environment list in the Context Switcher.
```

and I would leave the environment for now :D if there is a need for renaming, we will rename it later.